web/views/category_view.ex

```diff
@@ -2,7 +2,11 @@ defmodule CodeCorps.CategoryView do
   use CodeCorps.Web, :view
   use JaSerializer.PhoenixView
 
+  def preload(query, _conn, []) do
+    query |> CodeCorps.Repo.preload([:project_categories])
+  end
+
   attributes [:name, :slug, :description]
 
-  has_many :project_categories, serializer: CodeCorps.ProjectCategoryView
+  has_many :project_categories, serializer: CodeCorps.ProjectCategoryView, identifiers: :always
```

Comment:
Agreed. And it took me a long time to find it to, but it seems to be the convention. It doesn't matter if we do the other changes (moving of preloads). Identifiers do not load unless we specify it this way.